packages/react-native-otlp/package.json

```diff
@@ -31,5 +31,29 @@
   "embrace": {
     "iosVersion": "6.5.1",
     "androidVersion": "6.13.0"
-  }
+  },
+  "author": "Embrace <support@embrace.io> (https://embrace.io/)",
```

Comment:
building on iOS complains if these are missing

Reply:
although with it in place the linter fails since it isn't expected these attributes for a private package

Reply:
will remove the private key soon. I'll keep you posted